docs/src/pages/guides/composition/composition.md

```diff
@@ -26,21 +26,21 @@ WrappedIcon.muiName = Icon.muiName;
 
 {{"demo": "pages/guides/composition/Composition.js"}}
 
-## Component property
+## Component prop
 
-Material-UI allows you to change the root node that will be rendered via a property called `component`.
+Material-UI allows you to change the root element that will be rendered via a prop called `component`.
 
 ### How does it work?
 
 The component will render like this:
 
 ```js
-return React.createElement(this.props.component, props)
+return React.createElement(props.component, props)
 ```
 
 For example, by default a `List` component will render a `<ul>` element.
-This can be changed by passing a [React component](https://reactjs.org/docs/components-and-props.html#function-and-class-components) to the `component` property.
-The following example will render the `List` component with a `<nav>` element as root node instead:
+This can be changed by passing a [React component](https://reactjs.org/docs/components-and-props.html#function-and-class-components) to the `component` prop.
+The following example will render the `List` component with a `<nav>` element as root element instead:
```

Comment:
```suggestion
The following example will render the `List` component with a `<nav>` element as the root element instead of `ul`:
```
?

Reply:
Is this a good example? `<li>` aren't valid children of `<nav>` AFAIK

Reply:
Right, adding `button` to make it a `nav > button`.